lib/synapse/statsd.rb

```diff
@@ -0,0 +1,28 @@
+require 'statsd-ruby'
+require 'synapse/log'
+
+module Synapse
+  module StatsD
+    def statsd
+      @@STATSD ||= StatsD.statsd_for(self.class.name)
+    end
+
+    class << self
+      include Logging
+
+      @@STATSD_HOST = "localhost"
+      @@STATSD_PORT = 8125
+
+      def statsd_for(classname)
+        log.info "synapse: creating statsd client on host '#{@@STATSD_HOST}' port #{@@STATSD_PORT}"
```

Comment:
going to get a lot of identical log lines. maybe include `classname` in the log line? but this could probably be `log.debug`

Reply:
I'll do both recommendations.